src/rgw/setup/rgw.py

```diff
@@ -434,7 +422,8 @@ def _create_rgw_user(conf: MappedConf):
         # Adding retry logic for user creation with given timeout value.
         retry_count = 0
         rc = -1
-        while(retry_count < const.USER_CREATION_MAX_RETRY_COUNT):
+        while (retry_count < const.USER_CREATION_MAX_RETRY_COUNT):
+            Log.info(f'Creating admin user, command : {create_usr_cmd}.')
```

Comment:
no lets not log admin user creation command, its little sensitive info.

Reply:
Done